client/subscriptions-empty-state/style.scss

```diff
@@ -0,0 +1,31 @@
+.wcpay-empty-subscriptions__container {
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	align-items: center;
+	padding: 48px 0;
+	position: static;
+	height: 344px;
+	left: calc( 50% - 1240px / 2 );
+	top: 271px;
+	flex: none;
+	order: 1;
+	flex-grow: 0;
+	margin: 0;
+	text-align: center;
+	.wcpay-empty-subscriptions__description {
+		max-width: 535px;
+		margin: 15px 0;
+	}
+	.wcpay-empty-subscriptions__tos {
+		font-size: 11px;
+		color: #757575;
+		margin-bottom: 6px;
+	}
+	.wcpay-empty-subscriptions__button_container {
+		margin: 8px 0;
+		.is-secondary {
+			margin-left: 10px;
```

Comment:
**Extreme nitpick:** Possibly a little better to add the margin to the "Finish Setup" button instead as it is conditionally rendered.

By adding the margin to the "Create subscription product" button you end up with it looking slightly off-balance when the "Finish Setup" button is not present.

e.g.
<img width="539" alt="Screen Shot 2021-11-23 at 12 29 43 pm" src="https://user-images.githubusercontent.com/1527164/142959113-90411899-e118-4a11-8e60-3321f8f4c808.png">

Reply:
Oh, man. Matt was looking at this screen yesterday saying something looked off. How did I not notice that? 😆 

I submitted 40e31d7 which adds the margin to all buttons except the first one. It should have the same result but is more future proof. Let me know if that's ok.